src/services/AnalysisService.ts

```diff
@@ -219,19 +390,34 @@ class AnalysisService {
 
     if (contributingDeveloperAudit?.length === 0) {
       soosLogger.info(`Integration Name: ${integrationName}`);
-      const envVariableName = integrationNameToEnvVariable[integrationName];
-      if (envVariableName) {
-        const contributingDeveloper = process.env[envVariableName];
-        if (contributingDeveloper) {
+
+      // loop through all possible contributing developer environment variables and add the values for any that are found
+      for (const ev in contributingDeveloperEnvironmentVariables) {
```

Comment:
you can add a bool param `stopAtFirst` defaulted to true to that func above, then set it to false for the call here, so you don't have to repeat the code - check the bool on the loop iteration